### DISCUSSION TITLE
Is there an issue with the Firejail PPA?

### DISCUSSION THREAD

**Discussion by SuperSentient** [netblue30/firejail#5875] [Q&A] [answered]
I was using Kubuntu 22.04.2 today and when I checked for updates I got an error message saying something about the Firejail PPA. Is there an issue with the PPA or what? I've tried doing some research and I couldn't find any results.
Please help.

**Comment by ghost** [comment]
The Firejail PPA hasn't seen any changes after 2023-01-22, when firejail - 0.9.72-2~0ubuntu22.04.0 was released. Nothing related to the PPA has been reported in our issue tracker here recently, so it would be nice if you could elaborate on what error you see exactly. In the mean time we can ping @reinerh, the PPA's maintainer and collaborator here. If currently there's a problem he's best placed to get to the bottom of it.

**Reply by SuperSentient** [reply]
Well I only got the error message twice and I believe it said something like Firejail PPA no longer or supported. Or did it say no longer active?
Either way I only saw it twice heck even on my laptop which is running Linux Mint 21.1 with the FJ PPA I didn't see this error message.
Maybe it was a fluke/bug.

**Reply by ghost** [reply, marked as answer]
> Maybe it was a fluke/bug.

Might have been a temporary problem with `DNS resolution` of ppa.launchpad.net, or launchpad being down while you did an upgrade. It doesn't look like anything specific to Firejail so we can't actually do much here. I suggest to double-check your DNS setup and check your system logs when it happens again.

**Reply by SuperSentient** [reply]
:+1:
